app/components/global/AppBar/ProfileMenu.js

```diff
@@ -84,6 +84,18 @@ const MenuItemLink = styled(Link)`
     color: ${th('colorTextReverse')};
   }
 `
+const MenuItemExternalLink = styled.a`
```

Comment:
Let me pull this c ode down and try to assign this styling as a variable seeing as its duplicated for both the `Link` and `a`